### ISSUE TITLE
`tasmax` at frequency `1hr` should be removed from data request urbcc

### ISSUE THREAD

**Issue by larsbuntemeyer** [impetus4change/T32-CPRCM#12]
In [data-request-fpsurbrcc.csv](https://github.com/impetus4change/T32-CPRCM/blob/main/data-request-fpsurbrcc.csv), the *daily* temperature maxima and minima are requested at hourly frequency which does no make sense. I think it should be removed...

**Comment by jesusff** [comment]
Yes, good catch. This was inherited from the FPS Google sheet. I'm not sure if this was just a typo or these were meant to be hourly temperature maxima/minima

**Comment by LFEBoberg** [comment]
The same goes for wsgsmax, right?

**Comment by jesusff** [comment]
Thank you, yes. This is exactly the same situation as with tasmax/min. Hourly `wsgsmax` was in the original data request from the FPS-CONV, but defined without the "Daily" in the `long_name`. Probably intended as hourly maximum wind gust. The comments in the excel file says:
> can be defined as either the maximum wind speed of a time interval or calculated from a gust parameterization; indicate method in "comments"

The "Daily" part comes from the [CORDEX-CMIP6 data request](https://github.com/WCRP-CORDEX/data-request-table/blob/c6603fbcd8fe505b46107921c3c11a945131e999/CORDEX-CMIP6/data-request.csv#L353), where I took the rows for this CSV version of the FPS-CONV data request.

In summary, for this variable, we should either remove entirely the 1hr request or delete the "Daily" from the `long_name`
